src/Cache/Engine/RedisEngine.php

```diff
@@ -118,26 +132,48 @@ protected function _connect(): bool
         }
 
         try {
-            $this->_Redis = $this->_createRedisInstance();
-            if (!empty($this->_config['unix_socket'])) {
-                $return = $this->_Redis->connect($this->_config['unix_socket']);
-            } elseif (empty($this->_config['persistent'])) {
-                $return = $this->_connectTransient($tls . $this->_config['server'], $ssl);
+            if (!empty($this->_config['nodes'])) {
+                $this->_Redis = new RedisCluster(
```

Comment:
The code will be cleaner if you make 2 separate functions `connectRedis()` and `connectRedisCluster()` instead of the having the multiple if/else blocks.

Reply:
I have separated these connecting functions. The current naming conversion if the file suggested that these function names start with underscore so they are `_connectRedis()` and `_connectRedisCluster()`